handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslClientContext.java

```diff
@@ -63,12 +63,23 @@
                                          CipherSuiteFilter cipherFilter, ApplicationProtocolConfig apn,
                                          String[] protocols, long sessionCacheSize, long sessionTimeout,
                                          boolean enableOcsp) throws SSLException {
+        this(trustCertCollection, trustManagerFactory, keyCertChain, key, keyPassword,
```

Comment:
Since this is package-private, modify this constructor to add the new parameter instead of adding a new constructor.